lib/resources/r4/medication_request.yaml

```diff
@@ -131,6 +137,9 @@ fields:
     - display: RxNorm
       system: https://www.nlm.nih.gov/research/umls/rxnorm
       info_link: https://www.nlm.nih.gov/research/umls/rxnorm/index.html
+    - display: Cerner Medication Request Synonym's for orders
+      system: https://fhir.cerner.com/synonym
```

Comment:
Change the Details to Details: Millennium Medication Order Synonym
The System should be System:https://fhir.cerner.com/<EHR source id>/synonym

What is Details currently linking to?

Reply:
Addressed changed the system and details 
<img width="692" alt="Screenshot 2020-08-24 at 11 47 56 AM" src="https://user-images.githubusercontent.com/3060097/91010261-d0c1bb80-e5ff-11ea-85a3-e2fdd686ceff.png">